rspace/src/main/scala/coop/rchain/rspace/ScodecModels.scala

```diff
@@ -0,0 +1,60 @@
+package coop.rchain.rspace.scodecmodels
+
+import coop.rchain.rspace.Serialize
+import scodec.{Attempt, Codec, DecodeResult}
+import scodec.bits.{BitVector, ByteVector}
+import scodec.codecs.{bool, bytes, int32, listOfN, variableSizeBytes}
+
+private[rspace] case class BytesList(values: List[ByteVector])
```

Comment:
During development these types were helpful since there was harder to miss one List[ByteVector] with another. Now definitely they're just a waste of screen space. Refactored.